packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -113,67 +194,50 @@ export class SliderBase extends React.Component<ISliderProps, ISliderState> impl
+            </Label>
+          )}
+          <div {...sliderBoxProps}>
             <div ref={this._sliderLine} className={classNames.line}>
               {originFromZero && (
                 <span
                   className={css(classNames.zeroTick)}
-                  style={this._getStyleUsingOffsetPercent(vertical, zeroOffsetPercent)}
+                  style={this._getStyleUsingOffsetPercent(vertical, originPercent)}
+                />
+              )}
+              {ranged && (
+                <span
+                  ref={this._lowerValueThumb}
+                  className={classNames.thumb}
+                  style={this._getStyleUsingOffsetPercent(vertical, lowerThumbOffsetPercent)}
+                  {...lowerValueThumbProps}
                 />
               )}
               <span
                 ref={this._thumb}
                 className={classNames.thumb}
                 style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)}
+                {...thumbProps}
+              />
+              <span
+                className={css(classNames.lineContainer, classNames.inactiveSection)}
+                style={{ [lengthString]: bottomSectionWidth + '%' }}
+              />
+              <span
+                className={css(classNames.lineContainer, classNames.activeSection)}
+                style={{ [lengthString]: activeSectionWidth + '%' }}
+              />
+              <span
+                className={css(classNames.lineContainer, classNames.inactiveSection)}
+                style={{ [lengthString]: topSectionWidth + '%' }}
               />
-              {originFromZero ? (
-                <>
-                  <span
-                    className={css(classNames.lineContainer, classNames.inactiveSection)}
-                    style={{ [lengthString]: Math.min(thumbOffsetPercent, zeroOffsetPercent) + '%' }}
-                  />
-                  <span
-                    className={css(classNames.lineContainer, classNames.activeSection)}
-                    style={{ [lengthString]: Math.abs(zeroOffsetPercent - thumbOffsetPercent) + '%' }}
-                  />
-                  <span
-                    className={css(classNames.lineContainer, classNames.inactiveSection)}
-                    style={{ [lengthString]: Math.min(100 - thumbOffsetPercent, 100 - zeroOffsetPercent) + '%' }}
-                  />
-                </>
-              ) : (
-                <>
-                  <span
-                    className={css(classNames.lineContainer, classNames.activeSection)}
-                    style={{ [lengthString]: thumbOffsetPercent + '%' }}
-                  />
-                  <span
-                    className={css(classNames.lineContainer, classNames.inactiveSection)}
-                    style={{ [lengthString]: 100 - thumbOffsetPercent + '%' }}
-                  />
-                </>
-              )}
```

Comment:
This looks like a change in what's rendered that isn't guarded by a check that `ranged` is true. Can this be reworked so that the rendering does not change for non-`ranged` sliders?

Reply:
The code here is borrowed from v8 Slider component, which is not specific to range slider but a better way to render the slider track. It also makes the code clearer and more compact. I think it's worth to make the change here as a general improvement for the existing slider